apps/frontend/src/libs/components/meeting-form/meeting-form.tsx

```diff
@@ -0,0 +1,98 @@
+import { Button, Input } from "~/libs/components/components.js";
+import { CREATE_MEETING_FORM_DEFAULT_VALUES } from "~/libs/constants/constants.js";
+import { ButtonVariant } from "~/libs/enums/enums.js";
+import {
+	useAppDispatch,
+	useAppForm,
+	useAppSelector,
+	useCallback,
+} from "~/libs/hooks/hooks.js";
+import { type MeetingCreateRequestDto } from "~/libs/types/types.js";
+import {
+	actions as meetingActions,
+	meetingCreateValidationSchema,
+} from "~/modules/meeting/meeting.js";
+
+import styles from "./styles.module.css";
+
+type Properties = {
+	onClose: () => void;
+};
+
+const MeetingForm = ({ onClose }: Properties): React.JSX.Element => {
+	const dispatch = useAppDispatch();
+	const user = useAppSelector((state) => state.auth.user);
+	const { control, errors, handleSubmit } = useAppForm<MeetingCreateRequestDto>(
+		{
+			defaultValues: CREATE_MEETING_FORM_DEFAULT_VALUES,
+			validationSchema: meetingCreateValidationSchema,
+		},
+	);
+
+	const handleSubmitMeeting = useCallback(
+		async (data: MeetingCreateRequestDto): Promise<void> => {
+			if (!user) {
+				return;
+			}
+
+			const meetingData = {
+				...data,
+				ownerId: user.id,
```

Comment:
Backend shouldn't receive ownerId, it should set ownerId to the id of a user who sent a request. If this is not a case, please, fix it